source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -586,7 +594,7 @@ void Filter::onReceiveMessage(std::unique_ptr<ProcessingResponse>&& r) {
     stats_.stream_msgs_received_.inc();
     processing_complete_ = true;
     closeStream();
-    onFinishProcessorCalls();
+    onFinishProcessorCalls(absl::UnknownError(""));
```

Comment:
Can 'processing_status' be passed instead of absl::UnknownError here?

Reply:
Thanks, change made.